package/CHANGELOG

```diff
@@ -19,6 +19,8 @@ The rules for this file:
  * 2.4.0
 
 Fixes
+  * Add 'PairIJ Coeffs' to the list of sections in LAMMPSParser.py
```

Comment:
You will need to merge with upstream and move this to 2.5.0

Should be 

```
git fetch upstream
git merge upstream/develo
# fix any merge conflicts if they arise.
```

Reply:
Thanks, merge also done. Shall I do a push after the merge?

Reply:
Yes, just keep pushing to your branch and this PR.

Reply:
As per @hmacdope's comment - could you move this up to the 2.5.0 entry please ? You'll also need to add yourself to the user list there.